stdlib/universe/group.go

```diff
@@ -392,3 +393,24 @@ func (r MergeGroupRule) Rewrite(ctx context.Context, lastGroup plan.Node) (plan.
 
 	return merged, true, nil
 }
+
+// `OrderFilterGroup` orders to have 'filter |> group' from 'group |> filter'
+type OrderFilterGroup struct{}
```

Comment:
Could you change this name to `GroupFilterTransposeRule`. We have a similar rule that transposes group and window in `influxdb`, so it would be good to follow the convention.